istioctl/pkg/writer/envoy/configdump/testdata/configdump.json

```diff
@@ -0,0 +1,11131 @@
+{
```

Comment:
could we not check in 11k lines for this test? I think we only need a very small subset